modules/oncodrivefml/1.0/config/default.yaml

```diff
@@ -0,0 +1,70 @@
+lcr-modules:
+    
+    oncodrivefml:
+        
+        inputs:
+            # Available wildcards: {seq_type} {genome_build}
+            input_maf: "__UPDATE__"
+            subsetting_categories: "__UPDATE__"
+
+        genome_builds: ["__UPDATE__"]
+        # Available regions options: cds, lincrna, 3utr, 5utr, promoter_splice_sites
+        regions: ["__UPDATE__"]
+
+        maf_processing:
+            sample_sets: ["__UPDATE__"]
+            prepare_mafs: "{SCRIPTSDIR}/generate_smg_inputs/1.1/generate_smg_inputs.R"
+            include_non_coding: False
+            blacklists: []
+            blacklist_drop_threshold: 4
+
+        format_fml_input:
+            input_columns: "5,6,11,13,16"
+            additional_commands: ""
+
+        reference_files_directory: "ref/lcr-modules-references/downloads/oncodrive/"
+        regions_files:
+            grch37:
```

Comment:
maybe, it will make sense also to add these to the `etc` folder just like for hg38? How big are these files?

Reply:
The files are not very big at all. The biggest is about 2MB
https://bitbucket.org/bbglab/oncodrivefml/downloads/

I can move them to the etc folder, but they are currently set up to be downloaded through the reference files directory because they are used by both OncodriveCLUSTL and OncodriveFML. I can move the regions files into the `etc/` folder of both modules if that is preferred!